src/ITR-examples/utils.py

```diff
@@ -1,7 +1,8 @@
 import pandas as pd
 import numpy as np
 import copy as copy
-import random
+
+# import random
```

Comment:
Truly this is random.  I think this was put in place so that we could generate random company profiles, which I then decided not to do because it makes interpreting tests extra difficult (one must understand the random numbers).  So can delete.